tests/codegen/float/f128-algebraic.rs

```diff
@@ -0,0 +1,40 @@
+// Verify that algebraic intrinsics generate the correct LLVM calls for f128
```

Comment:
Could you actually remove the return type from the `CHECK-LABEL` annotations? `f16` and `f128` have a different ABI on different platforms that would need to be handled with something like https://github.com/rust-lang/rust/blob/10baf4465a241bedad114cd27288475febb96388/tests/codegen/float/f128.rs#L63-L66, but we don't need to add the complexity to verify that here.

Also it would be best to smoosh the four test files into a single `algebraic.rs` to save the testsuite a few compilations.

Tests look good though, thanks for adding them 👍

Reply:
Thanks! Done. Is it OK to keep the type in the internal `CHECK` lines since those aren't part of the ABI?

Reply:
Yeah that is good, it's just the function signatures that can change. Thanks!